tests/Unit/ProtoTests/Kms/out/samples/V1/KeyManagementServiceClient/get_crypto_key.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+require_once __DIR__ . '../../../vendor/autoload.php';
+
+// [START cloudkms_v1_generated_KeyManagementService_GetCryptoKey_sync]
+use Google\ApiCore\ApiException;
+use Google\Cloud\Kms\V1\CryptoKey;
+use Google\Cloud\Kms\V1\KeyManagementServiceClient;
+
+/**
+ * Returns metadata for a given [CryptoKey][google.cloud.kms.v1.CryptoKey], as
```

Comment:
This format is common in protos, but for PHP specifically we may be better off using [`@see`](https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/see.html) tags, as I believe it's more idiomatic. Also the proto name (in this case `google.cloud.kms.v1.CryptoKey`) is not really relevant.

```
 * Returns metadata for a given {@see CryptoKey}, as
```

Reply:
👍  - There is a todo here for something like this when possible, https://github.com/googleapis/gapic-generator-php/pull/483/files#diff-6d2583711a56ed5f6ed9262793b7185f0bc017a49a35580b8056429bb2d491fcR161